src/main/services/i18n/index.ts

```diff
@@ -21,4 +21,13 @@ i18next.use(Backend).init({
   }
 })
 
+// add language list here
+i18next.addResourceBundle('en', 'language', {
```

Comment:
In general, the previous solution looked logical, although the current one works, since it will just be a fallback to `en`. I do not even know what is more correct )

Reply:
I'm guessing this is because of the alternate mechanism.
This is handled by mapping, only "en" has this value, so just use it?
Maybe change it to something else and it will work fine.

Reminds me of a comic: it works fine, don't modify it.
![image](https://user-images.githubusercontent.com/17669599/180377289-c1fbdb68-0204-4816-a453-b6c4ccb1bc61.png)

Reply:
Did I misunderstand what you meant before? 😂

Reply:
I know exactly why it works, because there is a fallback to `en` if the selected locale has either no translation or no property at all. 

I'm just confused which option to leave just. Because they both work.

Reply:
Let's keep the first option, because even without knowing the logic of i18n, it looks more logical

Reply:
OK, done.
My git usage is not very good, and the network with github also fluctuates. It took some time.